webCommander/centralHub/xTwitter/twitterAPI.py

```diff
@@ -5,7 +5,7 @@
 import pprint 
 from time import sleep   
 from requests_oauthlib import OAuth1Session                                                                                                                                                
-from dotenv import load_dotenv, find_dotenv
+from dotenv import load_dotenv
```

Comment:
I like python-decouple better, specially when it comes to casting booleans (which we'll need for the DEBUG flag for example in settings.py)

I would also process all env variables in one place which would be settings.py

Then you can access them here via:

```
from django.conf import settings

settings.VAR
```